src/actions/AppStoreActions.js

```diff
@@ -343,6 +345,8 @@ export const changeCourseColor = (sectionCode, newColor, term) => {
     dispatcher.dispatch({
         type: 'COURSE_COLOR_CHANGE',
         addedCoursesAfterColorChange,
+        sectionCode,
+        newColor,
```

Comment:
Alternatively, in [AppStore](https://github.com/icssc/AntAlmanac/blob/color-fix/src/stores/AppStore.js#L189-L190), we could compare `this.addedCourses` to `action.addedCoursesAfterColorChange` to figure out what the changed color was/section code was. 

This doesn't require another emitter but it also would take longer to parse. I'm undecided on whether or not it's better...

I'm gonna think about this longer.

TBH we might want to rethink how we handle state management. Our current AppStore model is pretty confusing and clunky.